docker-compose.yml

```diff
@@ -261,3 +261,11 @@ services:
     volumes:
       - "./fossildb-dev/data:/fossildb/data"
       - "./fossildb-dev/backup:/fossildb/backup"
+
+  # Redis
+  redis:
+    image: redis:5.0.3
+    volumes:
+      - "./redis-data:/data"
```

Comment:
Not sure if we need this to be persisted on-disk for local development / CI, I would rather avoid it. If it should still stay in, let's use this command:
```
- "redis-server"
- "--appendonly"
- "yes"
```